integrations/artifactory_buildinfo.rst

```diff
@@ -0,0 +1,63 @@
+.. _integrations_artifactory_build_info:
+
+Artifactory Build Info
+======================
+
+The `Artifactory build info <https://www.buildinfo.org/>`_ Artifactory build info is a recollection of the metadata of a build.
+This json-formatted file includes all the details about the build broken down into segments like version history, artifacts, project
+modules, dependencies, and everything that was required to create the build.
+
+Build infos are identified with a ``build name`` and a ``build number``, similar to how many CI services identify the builds.
+They are conveninently stored in Artifactory to keep track of the build metadata and to perform different operations like merging
+the metadata with other build infos or build promotion between repositories.
+
+Conan has support for the build info at the `extesions repository <https://github.com/conan-io/conan-extensions>` using the feature
+:ref:`custom commands<reference_commands_custom_commands>`, that provides additional commands to create and manage the build info files
+uploaded to Artifactory.
+
+How to install the build info custom commands
+---------------------------------------------
+
+Using the dedicated repository for Conan extensions https://github.com/conan-io/conan-extensions, it is as easy as:
```

Comment:
Maybe a note to warn that this will not only install BuildInfo support but other extra things so people don't get surprised if suddenly several new commands appear?